### PR TITLE
Support for download from Azure

### PR DESCRIPTION
**Pull request by beutepa**
Support to download the CD and pre-requisites from Azure File or Blob storage.

### CODE REVIEW

File: Source/Modules/ISHServer/Get-ISHCD.ps1 [hunk 1]
```diff
@@ -42,8 +42,29 @@ function Get-ISHCD
...
+        [Parameter(Mandatory=$true,ParameterSetName="From Azure BlobStorage")]
+        [string]$ContainerName,
+        [Parameter(Mandatory=$true,ParameterSetName="From Azure BlobStorage")]
+        [ValidatePattern(".+\.[0-9]+\.0\.[0-9]+\.[0-9]+.*\.exe")]
+        [string]$BlobName,
+        [Parameter(Mandatory=$false,ParameterSetName="From Azure FileStorage")]
+        [Parameter(Mandatory=$false,ParameterSetName="From Azure BlobStorage")]
+        [string]$StorageAccountName,
+        [Parameter(Mandatory=$false,ParameterSetName="From Azure FileStorage")]
+        [parameter(ParameterSetName="From Azure BlobStorage")]
```
Comment (Sarafian) [inline Source/Modules/ISHServer/Get-ISHCD.ps1]: Need to explicitly defined the parameter's `Mandadory` attribute

File: Source/Modules/ISHServer/Private/Get-ISHAzureBlobObject.ps1 [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        }
+    }
+
+    process
+    {
+        $BlobName | ForEach-Object {
+            $localFile=Join-Path $LocalFolder ($_.Substring($_.LastIndexOf('/')+1))
+            Write-Debug "path=$_"
+            Write-Debug "localFile=$localFile"
+			if(-not (Test-Path $localFile) -or $Force)
```
Comment (Sarafian) [inline Source/Modules/ISHServer/Private/Get-ISHAzureBlobObject.ps1]: Improve indendation

File: Source/Modules/ISHServer/Private/Get-ISHAzureFileObject.ps1 [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+        }
+    }
+
+    process
+    {
+        $Path | ForEach-Object {
+            $localFile=Join-Path $LocalFolder ($_.Substring($_.LastIndexOf('/')+1))
+            Write-Debug "path=$_"
+            Write-Debug "localFile=$localFile"
+			if(-not (Test-Path $localFile) -or $Force)
```
Comment (Sarafian) [inline Source/Modules/ISHServer/Private/Get-ISHAzureFileObject.ps1]: Indendation

**Comment by Sarafian** [comment]
@beutepa are we keeping the `[object]Context` parameter?

**Comment by beutepa** [comment]
Let's keep it for now. 

The root commands Get-AzureStorageBlobContent/Get-AzureStorageFileContent need this Context. They don't support StorageAccountName/StorageAccountKey parameters.

I will investigate if there is a 'ProfileName' in Azure like in AWS S3 to align.
After the investigation, we can decide at which level we do the 'conversion' to the Context.

**Pull request merged**